README.md

```diff
@@ -2,139 +2,80 @@
-or generation. 
 
-#### Step 2: Create Other Tools, as needed
+2. **Create other tools (optional)**
 
 In addition to RAG tools, you can generate a lot of other types of tools the agent can use. These could be mathematical tools, tools 
-that call other APIs to get more information, and much more.
-
-`vectara-agentic` provides a few tools out of the box:
-1. Standard tools: 
-- `summarize_text`: a tool to summarize a long text into a shorter summary (uses LLM)
-- `rephrase_text`: a tool to rephrase a given text, given a set of rephrase instructions (uses LLM)
-  
-2. Legal tools: a set of tools for the legal vertical, such as:
-- `summarize_legal_text`: summarize legal text with a certain point of view
-- `critique_as_judge`: critique a legal text as a judge, providing their perspective
-
-3. Financial tools: based on tools from Yahoo Finance:
-- tools to understand the financials of a public company like: `balance_sheet`, `income_statement`, `cash_flow`
-- `stock_news`: provides news about a company
-- `stock_analyst_recommendations`: provides stock analyst recommendations for a company.
-
-4. database_tools: providing a few tools to inspect and query a database
-- `list_tables`: list all tables in the database
-- `describe_tables`: describe the schema of tables in the database
-- `load_data`: returns data based on a SQL query
+that call other APIs to get more information, or any other type of tool.
 
-You can create your own tool directly from a Python function using the `create_tool()` method:
+See [Tools](#agent-tools) for more information.
 
-```Python
-def mult_func(x, y):
-    return x*y
-
-mult_tool = ToolsFactory().create_tool(mult_func)
-```
-
-More tools coming soon!
- 
-#### Step 3: Create your agent
+3. **Create your agent**
 
 ```python
 agent = Agent(
-    tools = tools,
+    tools = [query_financial_reports],
     topic = topic_of_expertise,
     custom_instructions = financial_bot_instructions,
-    update_func = update_func
 )
 ```
-- `tools` is the list of tools you want to provide to the agent
+- `tools` is the list of tools you want to provide to the agent. In this example it's just a single tool.
 - `topic` is a string that defines the expertise you want the agent to specialize in.
-- `custom_instructions` is an optional string that defines special instructions to the agent
-- `update_func` is a callback function that will be called by the agent as it performs its task
-  The inputs to this function you provide are `status_type` of type AgentStatusType and 
-  `msg` which is a string.
+- `custom_instructions` is an optional string that defines special instructions to the agent.
 
-Note that the Agent type (`OPENAI` or `REACT`) is defined as an environment variables `VECTARA_AGENTIC_AGENT_TYPE`.
-
-For example, for a financial agent we can use:
+For example, for a financial agent we might use:
 
 ```python
 topic = "10-K financial reports",
```

Comment:
I believe this should be `topic_of_expertise` based on your last code chunk.

Reply:
yes.